src/components/projects/projectId/attributes/attributeId/AttributeCalculations.tsx

```diff
@@ -35,9 +34,23 @@ import { getLabelingTasksByProjectId, getProjectTokenization } from "@/src/servi
 import { getAttributeByAttributeId, updateAttribute } from "@/src/services/base/project-setting";
 import { Application, CurrentPage } from "@/submodules/react-components/hooks/web-socket/constants";
 import { VisitBricksButton } from "@/src/components/shared/bricks/VisitBricksButton";
+import LLMResponseConfig from "./LLMResponseConfig";
+import useDebounce from "@/submodules/react-components/hooks/useHooks/useDebounce";
+import useRefFor from "@/submodules/react-components/hooks/useRefFor";
+import { simpleDictCompare } from "@/submodules/javascript-functions/validations";
+import { LLM_CODE_TEMPLATE_EXAMPLES } from "./LLM/llmTemplates";
+import { capitalizeFirst } from "@/submodules/javascript-functions/case-types-parser";
 
 const EDITOR_OPTIONS = { theme: 'vs-light', language: 'python', readOnly: false };
 
+export const LLM_PROVIDER_OPTIONS = [
```

Comment:
we have `attribute-calculation-helper.ts` where this and the below const can be moved.

Reply:
repositioned